tic_tac_toe/board.py

```diff
@@ -0,0 +1,48 @@
+from tic_tac_toe.views import BoardDisplay
+
+
+class Board:
+    def __init__(self):
+        self.board = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+
+    def __getitem__(self, index):
+        return self.board[index]
+
+    def turn(self):
+        view = BoardDisplay()
+        token = self.current_player()
+
+        print("Please enter 1-9.")
+
+        view.display_board(self.board)
+
+        user_input = int(input()) - 1
+
+        if self.valid_move(user_input):
+            self.move(user_input, token)
+        else:
+            self.turn()
+
+    def move(self, selection, token="X"):
+        self.board[selection] = token
+
+    def turn_count(self):
+        return self.board.count("X") + self.board.count("O")
+
+    def current_player(self):
+        if self.turn_count() % 2 == 0:
+            return "X"
+        else:
+            return "O"
+
+    def position_taken(self, index):
+        if self.board[index] == "X" or self.board[index] == "O":
+            return True
+        else:
+            return False
```

Comment:
A Python style note: this could be simplified to
```python
def position_taken(self, index):
    return self.board[index] == "X" or self.board[index] == "O"
```
(Also applies to the function below this one.) Since your expressions in line 39 all return booleans already